oct/cli/package/aws.py

```diff
@@ -30,25 +35,45 @@
     is_flag=True,
     help='Mark an AMI created previously as ready.',
 )
+@option(
+    '--tag',
+    '-t',
+    "tag",
```

Comment:
Err -- this changes the name of the variable used in the function, not the name of the flag